src/Client/Shared/Components/FileCard.razor

```diff
@@ -1,7 +1,7 @@
 ﻿@using Functionland.FxFiles.Client.Shared.Utils
 @inherits AppComponentBase;
 
-<div class="file-card-container @(IsPinned ? "favorite" : "") @(IsEnable ? "disabled-card" : "")">
+<div class="file-card-container @(true ? "favorite" : "") @(IsDisabled ? "disabled-card" : "")">
```

Comment:
what was the reason for changing this condition to `true`?

Reply:
Thanks. I fixed that